work/1 Table.sql

```diff
@@ -0,0 +1,17 @@
+use costumeDB
+go
+drop table if exists Costumes 
+go
+create table dbo.Costumes(
+CostumeId int not null identity primary key,
+CustomerName varchar (50) not null constraint ck_costumes_customername_cannot_be_blank check(customername <> ''),
```

Comment:
-- LL customer name column should be divided into two columns: First Name and Last Name